specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-11-01-preview/DynamicSummaries.json

```diff
@@ -0,0 +1,367 @@
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Creates or updates a Summary.": {
+            "$ref": "./examples/dynamicSummaries/CreateSummary.json"
+          },
+          "Creates or updates a summary and bulk creates summary items.": {
+            "$ref": "./examples/dynamicSummaries/CreateSummaryAndSummaryItems.json"
+          }
+        },
+        "tags": [
+          "Summaries"
+        ],
+        "description": "Create or update a Summary and its Summary Items (bulk creation). To create a Summary and its Items, we should call this endpoint with rawContent property. The rawContent is mainly used for small summary (content size below 3.8 MB). The status of processing such large file can be polled through the URL returned in Azure-AsyncOperation header.",
+        "operationId": "Summaries_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SummaryId"
+          },
+          {
+            "$ref": "#/parameters/Summary"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Summary"
+            }
+          },
+          "201": {
+            "description": "Created. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field.",
+            "schema": {
+              "$ref": "#/definitions/Summary"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "Contains the status URL on which clients are expected to poll the status of the operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SummaryList": {
+      "description": "List all the summaries.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
```

Comment:
>nextLink [](http://example.com/codeflow?start=9&length=8)

nextLink is already on the wrapper

Reply:
removing it results in error when running autorest validation. the x-ms-pageable requires to have nextLink defined. looking at other *.json (other sendpoints) in the same directory, they all defined the nextLink.